instance/instance_test.go

```diff
@@ -173,7 +173,8 @@ func (s *ManagerTestSuite) TestStartAndUpdate(t *C) {
 		{Code: 404, Data: nil, Error: nil},
 	}
 	in, err = m.Repo().Get("notfound", false)
-	t.Check(err, Equals, instance.ErrInstanceNotFound)
+	t.Check(err, FitsTypeOf, instance.ErrInstanceNotFound(""))
+	t.Check(err, ErrorMatches, "Cannot read instance file: .*/instance/notfound.json: open .*/instance/notfound.json: no such file or directory")
```

Comment:
Type of the error got changed to standard produced by `fmt.Errorf` to allow passing variables in string. I've changed the type again, so now it's again specific type `ErrInstanceNotFound` but also allows custom string.